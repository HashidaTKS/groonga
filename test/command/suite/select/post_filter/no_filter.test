table_create Items TABLE_NO_KEY
column_create Items price COLUMN_SCALAR UInt32

load --table Items
[
{"price": 100},
{"price": 200},
{"price": 300}
]

select Items \
  --post_filter "price >= 200"
