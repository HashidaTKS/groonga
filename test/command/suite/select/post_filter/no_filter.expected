table_create Items TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Items price COLUMN_SCALAR UInt32
[[0,0.0,0.0],true]
load --table Items
[
{"price": 100},
{"price": 200},
{"price": 300}
]
[[0,0.0,0.0],3]
select Items   --post_filter "price >= 200"
[[0,0.0,0.0],[[[2],[["_id","UInt32"],["price","UInt32"]],[2,200],[3,300]]]]
